pkg/catalogv2/helm/helm3.go

```diff
@@ -15,10 +19,14 @@ import (
 	"helm.sh/helm/v3/pkg/release"
 )
 
+// isHelm3 checks if the value of the owner key of the received map is equal to helm.
+// Every helm3 release object has this particular label on it.
 func isHelm3(labels map[string]string) bool {
 	return labels["owner"] == "helm"
 }
 
+// fromHelm3Data receives a helm3 release data.
```

Comment:
// fromHelm3Data receives a helm3 release data string of a helm chart installed.
// It then converts the string into Helm3 release struct and again to rancher
// v1.ReleaseSpec struct to return it.